examples/java/CPU.java

```diff
@@ -0,0 +1,80 @@
+import com.grafana.foundation.cog.Builder;
+import com.grafana.foundation.common.StackingConfig;
+import com.grafana.foundation.common.StackingMode;
+import com.grafana.foundation.dashboard.Panel;
+import com.grafana.foundation.dashboard.Threshold;
+import com.grafana.foundation.dashboard.ThresholdsConfig;
+import com.grafana.foundation.dashboard.ThresholdsMode;
+
+import java.util.List;
+
+public class CPU {
+    public static Builder<Panel> cpuUsageTimeseries() {
+        String query = "((1 - sum without (mode) (rate(node_cpu_seconds_total{job=\"integrations/raspberrypi-node\", mode=~\"idle|iowait|steal\", instance=\"$instance\"}[$__rate_interval]))) " +
+                "/ ignoring(cpu) group_left " +
+                "count without (cpu, mode) (node_cpu_seconds_total{job=\"integrations/raspberrypi-node\", mode=\"idle\", instance=\"$instance\"}))";
+
+        Threshold th1 = new Threshold();
```

Comment:
We should provide constructors for classes (not required now, an issue *for later*™ is fine)